src/backend/distributed/commands/foreign_constraint.c

```diff
@@ -116,21 +116,22 @@ ConstraintIsAForeignKeyToReferenceTable(char *constraintName, Oid relationId)
  *   table is not a reference table.
  */
 void
-ErrorIfUnsupportedForeignConstraintExists(Relation relation, char referencingDistMethod,
+ErrorIfUnsupportedForeignConstraintExists(Relation referencingRelation,
+										  char referencingDistMethod,
 										  Var *referencingDistKey,
 										  uint32 referencingColocationId)
 {
 	ScanKeyData scanKey[1];
 	int scanKeyCount = 1;
 
-	Oid referencingTableId = relation->rd_id;
+	Oid referencingRelationOid = referencingRelation->rd_id;
 	bool referencingNotReplicated = true;
-	bool referencingIsCitus = IsCitusTable(referencingTableId);
+	bool referencingIsCitusTable = IsCitusTable(referencingRelationOid);
```

Comment:
By looking at the code it seems that we use referencing and referenced prefixes to distinguish them.

The word `referencingIsCitusTable` sounds a bit strange to me. Because it could also mean 'we are referencing a citus table`. `referencing_isCitusTable` would be more clear but that breaks our naming convention. Not sure what naming could be better here.

Reply:
> By looking at the code it seems that we use referencing and referenced prefixes to distinguish them.

yes

> Because it could also mean we are referencing a citus table
I guess I do not feel the same completely, in that case, variable name would be `referencingToCitusTable` ?

Reply:
Yeah I guess by looking at the code, it becomes clear what they mean.